go-packages/meep-data-model/api/swagger.yaml

```diff
@@ -20,7 +20,28 @@ definitions:
       cellId:
         type: string
         description: The E-UTRAN Cell Identity as defined in ETSI TS 136 413 including the ID of the eNB serving the cell
-    description: Cellular POA configuration information
+    description: '**DEPRECATED** As of release 1.6.0, renamed to poa4GConfig'
+  Poa4GConfig:
+    type: object
+    properties:
+      cellId:
+        type: string
+        description: The E-UTRAN Cell Identity as defined in ETSI TS 136 413 including the ID of the eNB serving the cell
+    description: Cellular 4G POA configuration information
+  Poa5GConfig:
+    type: object
+    properties:
+      cellId:
+        type: string
+        description: The E-UTRAN Cell Identity as defined in ETSI TS 136 413 including the ID of the NR serving the cell
+    description: Cellular 5G POA configuration information
+  PoaWifiConfig:
+    type: object
+    properties:
+      macId:
+        type: string
+        description: TBD
+    description: WIFI POA configuration information
   CellularDomainConfig:
```

Comment:
I believe it is possible to have both 4G and 5G for one domain. But i believe the defaultCellId should actually later be removed. It's only use right now is because there was an issue in RNIS v1 where it was not handling well a notion where an edge is not necessarily connected to an antenna. It was something mandatory in v1, but it is fixed in v2. Since we intend to implement v2, the use of this will not be useful anymore, and as such will not be relevant and will be removed at this time. There is then no need to do something for a default nr_cell_id (5G).